atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/timestamp/TimestampServiceTests.java

```diff
@@ -100,16 +97,18 @@ public static void willDoNothingWhenFastForwardToEarlierTimestamp(
         Assertions.assertThat(ts2).isGreaterThan(ts1);
     }
 
-    public static void canReturnManyUniqueTimestampsInParallel(TimestampService timestampService)
+    public static void canReturnManyUniqueTimestampsInParallel(
+            TimestampService timestampService,
+            ExecutorService executor)
```

Comment:
note that we won't have to pass this in if this becomes an Abstract Base test class.